content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,97 @@
+# Data Management with DVC
+
+DVC helps you manage and share arbitrarily large files, datasets, and ML models
+anywhere: mounted drives, network resources (e.g. NAS), external devices, or
+remotely (SSH servers, cloud storage, etc.). Once the project is configured, you
+can manipulate files normally in your local workspace. DVC tracks, restores, and
+synchronizes them across locations.
+
+![Storage locations](/img/storage-locations.png) _Local, external, and remote
+storage locations_
+
+## The data cache
```

Comment:
I'm not sure this should go (at least in this form) into the index page.

I would still prefer one page, no subtitles initially, one story that highlights DVC's workflow with data, building blocks (no need for a long section per each block to my mind).

We need to highlight things like:

- Before: files are scattered in the Cloud, names are used to version, everyone can write/read, etc. Ad-hoc reproducibility.
- After: object in the cloud are organized and you are not supposed to touch them. Everything is happening though Git (optionally in cloud versioning etc can be accessed directly). Saving into git the list of files essentially that are being used to train the model guarantees reproducibility, etc 

I would think into this direction for the whole intro.

Reply:
OK this makes sense. I also felt that this index is a bit deep and somewhat similar to #4053's [Track, Sync, and Version Data](https://dvc-org-guide-data-mgmt-yguzqx.herokuapp.com/doc/user-guide/data-management/track-sync-version).

Reply:
> one page, no subtitles initially, one story that highlights DVC's workflow with data, building blocks

OK: for now I flattened & shortened the index page (except for the Versioning section) but it's still about the parts of DVC that enable its data management approach (local, external, remote locations // workspace, cache, remotes) -- not sure that's what you meant by "building blocks" @shcheklein 

> We need to highlight things like...

Have not addressed this yet... ⏳